src/clientManager.js

```diff
@@ -5,17 +5,25 @@ var colors = require("colors/safe");
 var fs = require("fs");
 var Client = require("./client");
 var Helper = require("./helper");
-var Oidentd = require("./oidentd");
 
 module.exports = ClientManager;
 
 function ClientManager() {
 	this.clients = [];
+}
+
+ClientManager.prototype.init = function(identHandler, sockets) {
+	this.sockets = sockets;
+	this.identHandler = identHandler;
+
+	if (!Helper.config.public) {
+		if ("autoload" in Helper.config) {
+			log.warn(`Autoloading users is now always enabled. Please remove the ${colors.yellow("autoload")} option from your configuration file.`);
+		}
 
-	if (typeof Helper.config.oidentd === "string") {
-		this.identHandler = new Oidentd(Helper.config.oidentd);
+		this.autoloadUsers();
 	}
-}
+};
```

Comment:
What's the reason behind moving autoloading users from `server.js` to `clientManager.js`?